conn.go

```diff
@@ -583,4 +583,17 @@ type Config struct {
 	// CAP_SYS_ADMIN are required), and most applications should leave this set
 	// to 0.
 	NetNS int
+
+	// DisableNSLockThread
+	//
+	// By default, the library will lock the processing go routine to its corresponding
+	// thread in order to enable communication over netlink to a different namespaces.
+	// In some cases, where the caller already knows that the netlink socket
+	// is in the same namespace as the calling thread, this can introduce a performance
+	// impact. In these cases, the caller can disable thread locking if performance
+	// consideration are of interest. If disabled, it is the responsibility of
+	// the caller to make sure that all threads are running in the correct namespace.
+	// When DisabeNSLockThread is set, the caller cannot set the NetNS value. An error
```

Comment:
```suggestion
	// When DisableNSLockThread is set, the caller cannot set the NetNS value. An error
```

Reply:
IMO just remove the last "An error will be returned" sentence.

Reply:
good point